.nvmrc

```diff
@@ -0,0 +1 @@
+v10.18.1
```

Comment:
This will not do anything if you haven't installed nvm. I'd rather suggest to set `engine-strict` as described [here](https://stackoverflow.com/a/50798589)

Reply:
Good call thanks,
I think some instructions on nvm are missing as this assumes we have nvm installed on the machine, we will still have to be doing `nvm use` to set it.
I was a bit reluctant to set it to strict as this could work for a range of certain versions of node possible from 8 to 11 (I could play with it to check), we could have a range instead which will still warn the user.
Happy to change to a stricter version if it makes sense and make it throws errors also we will have to check the impact it could have in the pipeline.
@m-ujaffer @castancu thoughts?